libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -114,16 +114,16 @@ class AC_AutoTune {
     void announce_state_to_gcs();
     void do_gcs_announcements();
 
-    enum LEVEL_ISSUE {
-        LEVEL_ISSUE_NONE,
-        LEVEL_ISSUE_ANGLE_ROLL,
-        LEVEL_ISSUE_ANGLE_PITCH,
-        LEVEL_ISSUE_ANGLE_YAW,
-        LEVEL_ISSUE_RATE_ROLL,
-        LEVEL_ISSUE_RATE_PITCH,
-        LEVEL_ISSUE_RATE_YAW,
+    enum struct LevelIssue {
```

Comment:
an "enum struct" .. this will be the first time we use this in AP as far as I know.  I guess the good thing about it is that it must be prefixed by the struct name so it doesn't pollute the rest of the namespace.